code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -281,7 +281,7 @@ In most cases it makes more sense to use apply_damage() instead! And make sure t
 This function restores the subjects blood to max.
 */
 /mob/living/carbon/human/proc/restore_blood()
-	if(!species.anatomy_flags & NO_BLOOD)
+	if(!(species.anatomy_flags & NO_BLOOD))
```

Comment:
Pretty sure this proc wouldn't have worked on any `species` with any `anatomy_flags`. However this proc is only used by random artifacts, so it didn't really matter.

Reply:
more specifically by darkrevive which is one of the most cumbersome artifacts to make use of. I should rework that effect someday.